config/Config-template.groovy

```diff
@@ -167,6 +167,20 @@ environments { development { com.recomdata.guestAutoLogin = true } }
 // control the level of access anonymous users will have (the access will be match
 // that of the account specified here).
 com.recomdata.guestUserName = 'guest'
+
+/*
+user.password.strength.regex = '^' +
+//Ensure string has an uppercase letter.
+'(?=.*[A-Z])' +
+//Ensure string has one special case letter.
+'(?=.*[!@#$&*])' + 
+//Ensure string has a digit.
+'(?=.*[0-9])' +
+//Ensure string length is not less then 8.
+'.{8,}' +
+'$'
+*/
```

Comment:
Instead of using concatenation, use a multine groovy string and enable the COMMENTS option with `(?x)`.